src/ui.js

```diff
@@ -144,16 +223,23 @@ async function promptConfirmDelete(repoCount) {
   });
 }
 
-function printConfirmDelete(deletedRepos) {
-  const count = deletedRepos.length;
-
-  const strDeletedRepos = count > 1 ? deletedRepos.join(', ') : deletedRepos;
-  const strRepos = count > 1 ? 'repositories' : 'repository';
-  const strConfirm = `🔫 pew pew! ${count} ${strRepos} deleted successfully: ${strDeletedRepos}`;
+function printConfirmDelete(deletedItems, type) {
+  const count = deletedItems.length;
+
+  const strDeletedItems = count > 1 ? deletedItems.join(', ') : deletedItems;
+  const strItems =
+    count > 1
+      ? type === 'repos'
+        ? 'repositories'
+        : 'codespaces'
+      : type === 'repos'
+      ? 'repository'
+      : 'codespace';
```

Comment:
Should this be a `switch` statement?

Reply:
In case I use a `switch` statement, it would be something like this (not proper syntax) - 
```pseudocode
switch count {
if it is greater than 1: {
if type = repos {
return repositories
} else {
return codespaces
}
similarly the singular words....
}
```

Using ternary makes it shorter, but I can use `switch` and `if else` if that would be better